File: web/src/api/memLogs.ts
import { http,resData } from "@/utils/http";
 
// @Tags MemLogs
// @Summary 创建MemLogs
export const createMemLogs = (data?: object) => {
   return http.request<resData>("post", "/memLogs/createMemLogs", { data });  
}

// @Tags MemLogs
// @Summary 删除MemLogs
export const deleteMemLogs = (data?: object) => {
   return http.request<resData>("post", "/memLogs/deleteMemLogs", { data });  
}
 

// @Tags MemLogs
// @Summary 删除ByIdsMemLogs
export const deleteMemLogsByIds = (data?: object) => {
   return http.request<resData>("post", "/memLogs/deleteMemLogsByIds", { data });  
}
 

// @Tags MemLogs
// @Summary 更新MemLogs
export const updateMemLogs = (data?: object) => {
   return http.request<resData>("post", "/memLogs/updateMemLogs", { data });  
}

// @Tags MemLogs
// @Summary 快速编辑
export const quickEdit = (data?: object) => {
   return http.request<resData>("post", "/memLogs/quickEdit", { data });  
}
 

// @Tags MemLogs
// @Summary 用id查询MemLogs
export const findMemLogs = (params?: object) => {
   return http.request<resData>("get", "/memLogs/findMemLogs", { params });  
}
 

// @Tags MemLogs
// @Summary 分页获取MemLogs列表
export const getMemLogsList = (params?: object) => {
   return http.request<resData>("get", "/memLogs/getMemLogsList", { params });  
}
 

// @Tags MemLogs
// @Summary 导出excelMemLogs列表
export const excelList = (params?: object) => {
   return http.request<resData>("get", "/memLogs/excelList", { params });  
}

File: server/router/business/memLogs.go
package business

import (
	"go-cms/api/v1"
	"go-cms/middleware"
	"github.com/gin-gonic/gin"
)

type MemLogsRouter struct {
}

// InitMemLogsRouter 初始化 MemLogs 路由信息
func (s *MemLogsRouter) InitMemLogsRouter(Router *gin.RouterGroup) {
	router := Router.Group("memLogs").Use(middleware.OperationRecord())
	routerNoRecord := Router.Group("memLogs")
	var apiV1 = v1.ApiGroupApp.BusinessApiGroup.MemLogsApi
	{
		router.POST("createMemLogs", apiV1.CreateMemLogs)   // 新建MemLogs
		router.POST("deleteMemLogs", apiV1.DeleteMemLogs) // 删除MemLogs
		router.POST("deleteMemLogsByIds", apiV1.DeleteMemLogsByIds) // 批量删除MemLogs
		router.POST("updateMemLogs", apiV1.UpdateMemLogs)    // 更新MemLogs
	    router.POST("quickEdit", apiV1.QuickEdit)  // 快速编辑	
	}
	{
		routerNoRecord.GET("findMemLogs", apiV1.FindMemLogs)        // 根据ID获取MemLogs
		routerNoRecord.GET("getMemLogsList", apiV1.GetMemLogsList)  // 获取MemLogs列表
        routerNoRecord.GET("excelList", apiV1.ExcelList)  // 分页导出excel MemLogs列表
	}
}
